samples/scheduling/calendar/size/src/index.tsx

```diff
@@ -1,13 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import { IgrCalendar, IgrRadioGroup, IgrRadio, IgrCalendarModule, IgrRadioGroupModule, IgrRadioModule } from 'igniteui-react';
+import { IgrCalendar, IgrRadioGroup, IgrRadio } from 'igniteui-react';
 import 'igniteui-webcomponents/themes/light/bootstrap.css';
 
-IgrCalendarModule.register();
-IgrRadioGroupModule.register();
-IgrRadioModule.register();
-
 export default class CalendarSize extends React.Component<any, any> {
```

Comment:
The IgrRadio for the large size is explicitly always selected because of the ```checked={true}```:

```
 <IgrRadio name="size" value="large" checked={true} onChange={this.onRadioChange}>
      <span>Large</span>
  </IgrRadio>
```
It should be bound to a state so that the user can actually change the selection of the radio. Currently it stays stuck on Large no matter which one you select.